style.css

```diff
@@ -0,0 +1,135 @@
+:root{
+    --border-color:#d1d1d1;
+    --table-element-color:#fb923c;
+    --primary-color: #94a3b8;
+    }
+
+/*cart-page*/
+
+
+.navbar{
+    /* position: sticky; */
+    top: 0;
+    left: 0;
+    width: 100%;
+    /* z-index: 9; */
+}
+
+.nav{
+    padding: 2px 2vw;
+    display: flex;
+    justify-content: space-between;
+}
+
+.nav-items{
+    display: flex;
+    align-items: center;
+}
+
+.brand-logo-cart{
+    margin-left: 35rem;
+    height:23rem;
+    width:23rem;
+}
+
+.nav-items a{
+    margin-left: 20px;
+}
+
+.nav-items a i{
+    width: 30px;
+}
+
+.links-container{
+    width: 100%;
+    display: flex;
+    padding: 10px 10vw;
+    justify-content: center;
+    list-style: none;
+    border-top: 1px solid var(--border-color);
+}
+.link{
+    text-transform: capitalize;
+    padding: 0 10px;
+    margin: 0 5px;
+    text-decoration: none;
+    color: #383838;
+    opacity: 0.5;
+    transition: .5s;
+}
+
+.cartpage-title{
+    font-size: 20px;
+    color: var(--primary-color);
+    
+    }
+
+.cart-page{
+    margin: 80px 20px;
+}
+
+table{
+     display:flexbox;
+     justify-content:center;  
+    
+   width:100%;
+   border-collapse: collapse; 
+}
+
+.cart-info{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+th{
+    text-align: left;
+    padding: 5px;
+    color:#fff;
+    background: var(--table-element-color);
+    font-weight: normal;   
+}
+
+td{
+    padding: 10px 5px;
+}
+
+td .cart-info{
+    padding: 10px 5px;
+}
+
+td input{
+
+    width:40px;
+    height: 30px;
+    padding:5px
+}
+
+td a{
+    color:var(--table-element-color);
+    font-size: 12px;
+}
+
+td img{
```

Comment:
Avoid directly applying style on image instead put image in a container and then apply style to that container.